sdks/python/apache_beam/coders/coders_test_common.py

```diff
@@ -40,10 +44,11 @@
 class CustomCoder(coders.Coder):
 
   def encode(self, x):
-    return str(x+1)
+    x = x + 1
+    return int(x).to_bytes((x.bit_length() + 7) // 8, 'big', signed=True)
```

Comment:
<!--new_thread; commit:57a73055b9447e7a1fe6ad674673254d9a614eeb; resolved:0-->
What is the rationale for changing the coder behavior here to use a binary encoding instead of using a printable decimal string, for what is intended to be a simple example?  Is there something simpler we can do? (e.g. we could coerce to `str` and encode as `.encode('latin-1')`)